ga4gh/datarepo.py

```diff
@@ -320,28 +324,73 @@ def close(self):
         self._dbConnection.close()
         self._dbConnection = None
 
+    def verify(self):
+        """
+        Verifies that the data in the repository is consistent.
+        """
+        # TODO this should emit to a log that we can configure so we can
+        # have verbosity levels. We should provide a way to configure
+        # where we look at various chromosomes and so on. This will be
+        # an important debug tool for administrators.
+        for referenceSet in self.getReferenceSets():
+            print(
+                "Verifying ReferenceSet", referenceSet.getLocalId(),
+                "@", referenceSet.getDataUrl())
+            for reference in referenceSet.getReferences():
+                length = min(reference.getLength(), 1000)
+                bases = reference.getBases(0, length)
+                assert len(bases) == length
```

Comment:
Hah, basically just to keep flake8 from complaining! I can get rid of it if it seems confusing, I just thought it was a useful piece of paranoia.